packages/astro/README.md

```diff
@@ -25,6 +25,18 @@ npm install
 npm start
 ```
 
+### pnpm
+
+If you are using [pnpm](https://pnpm.io/) add the following `.npmrc` to your root folder before installing:
+
+```
+shamefully-hoist = true
+```
+
+### Yarn
+
+[Yarn](https://yarnpkg.com/) works as a package manager in Astro apps. However, __Yarn 2__ changes how module resolution works in Node apps and doesn't support modules written in ESM at the moment. Since Astro is written entirely in ESM, you can't use Yarn 2 in Astro apps. We'll continue to track Yarn 2 as they fix these core bugs.
```

Comment:
Technically Yarn 2 has no issue with ESM, it's Yarn PnP that is missing support for it

Reply:
Good point! If you want to clarify this please send a PR.

Reply:
> Yarn 2 has PnP by default, though

That's true but Yarn 1, Yarn 2, Yarn 3 (currently in RC), and pnpm support installing using Yarn PnP so it might seem like it doesn't work with Yarn 2 only but the rest is fine. Since all versions of Yarn supports creating a `node_modules` layout it's a bit misleading. It doesn't bother me enough to change it, just wanted to provide some context.